tcl/org.testeditor.tcl.model/model/tcl.xcore

```diff
@@ -51,8 +51,21 @@ class TestStep {
 	contains StepContent[0..*] contents
 }
 
+class TestStepAssertion extends TestStep {	
```

Comment:
I would prefer `AssertionTestStep`, this is less ambiguous. It is a test step, that performs an assertion.